### PR TITLE
Make RollingFile filePattern an absolute path

### LINKED ISSUE #5883: Logrotation tries to write into CATALINA_HOME instead of the log directory
# Problem description

In our Tomcat logs we currently see a lot of these error messages:
```
ajp-nio-127.0.0.1-8009-exec-303 ERROR Unable to copy file /var/log/rhn/rhn_web_api.log to /usr/share/tomcat/rhn_web_api-1.log: java.nio.file.AccessDeniedException /usr/share/tomcat/rhn_web_api-1.log
```

## Version of Uyuni Server and Proxy (if used)

2022.08 on OpenSUSE 15.4

## Details about the issue

https://github.com/uyuni-project/uyuni/blob/b8042807d6f7297a734b86fd97098f2801f9066c/java/code/src/log4j2.xml#L9-L13

The Log4J2 RollingAppender tries to rotate the logfile because it is larger then the configured maximal size. But because the filePattern is relative it tries to write the rotated file into CATALINA_HOME - what is denied

### PR DESCRIPTION
**Pull request by mayrstefan**
## What does this PR change?

Should fix #5883. Log4J2 should no more try to write the rotated logfile into CATALINA_HOME

## GUI diff

No difference.

## Documentation
- No documentation needed: only internal and user invisible changes

## Test coverage
- No tests: no functional change

## Links

Fixes #5883

- [x] **DONE**

## Changelogs

- [x] No changelog needed (what is the criteria to make a fix worth a changelog entry?)

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by cbosdo — APPROVED** [review]

**Comment by mayrstefan** [comment]
What can be done to get the second review?

**Review by mcalmer — APPROVED** [review]

**Pull request merged**
